### DISCUSSION TITLE
Chias not send help

### DISCUSSION THREAD

**Discussion by JoeDaPro12** [Chia-Network/chia-blockchain#11284] [Support]
Wanted to send my chias to the exchange.
Output is not listed in my blockchain. I was always synced. But I still have 1.2.9 maybe because of the version?

**Comment by loppefaaret** [comment]
Even a 1.2.9 client should be able to send a transaction. perhaps you needed to add a hundred mojo's as a fee ?

**Comment by JoeDaPro12** [comment]
thanks that helps.
________________________________
Von: loppefaaret ***@***.***>
Gesendet: Montag, 25. April 2022 18:42
An: Chia-Network/chia-blockchain ***@***.***>
Cc: JoeDaPro12 ***@***.***>; Author ***@***.***>
Betreff: Re: [Chia-Network/chia-blockchain] Chias not send help (Discussion #11284)


Even a 1.2.9 client should be able to send a transaction. perhaps you needed to add a hundred mojo's as a fee ?
